azure_pipelines/scripts/get_automation_conf_file.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import base64
+import os
+import json
+
+def run_az_command(command):
+    try:
+        result = subprocess.run(command, shell=True, capture_output=True, text=True)
+        if result.returncode == 0:
+            return result.stdout.strip()
+        else:
+            print("Error:", result.stderr.strip())
+            return None
+    except Exception as e:
+        print("Exception:", str(e))
+        return None
+# 2 Loading conf.json file as base64 encoding form lab keyvault
+print("Loading conf.json file...")
+output_1 = ""
+output_2 = ""
+# 2.1 Loading the 1st half and formatting 
+az_command_1 = "az keyvault secret show --name \"appleautomationconfjsonfile1\" --vault-name \"buildautomation\" --query \"value\""
+output_1_without_formatting = run_az_command(az_command_1)
+output_1 = output_1_without_formatting.replace('\n', '').replace('"', '')
+# 2.2 Loading the 2nd half and and formatting
+az_command_2 = "az keyvault secret show --name \"appleautomationconfjsonfile2\" --vault-name \"buildautomation\" --query \"value\""
+output_2_without_formatting = run_az_command(az_command_2)
+# 3.1 Combine 2 parts, decode and write into local conf.json file
+output_2 = output_2_without_formatting.replace('\n', '').replace('"', '')
+base64_encoded_conf_file = output_1 + "" + output_2

```

Comment:
I don't remember why we base64 the file. Can we just add conf.json in the key vault without b64 encoding it?

Reply:
We store as 2 secrets (too long for 1 secret and will be rejected), and I think a simple encoded string would be easier for maintaining. Please feel free to explore the json approach, for now, we are using this script to checkout both locally and remotely